keel-core/src/main/kotlin/com/netflix/spinnaker/keel/notifications/NotificationMessage.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.notifications
+
+data class NotificationMessage(
```

Comment:
is this structure limited to a specific type of notification?

Reply:
No, it's the same for both email and slack